CBSE.py

```diff
@@ -197,6 +217,10 @@ def get_own_required_component_names(self):
         #TODO
         return ['r_' + c.get_type_name() for c in self.requiredComponents]
 
+    def get_own_further_dependencies(self):
```

Comment:
Not really related to this PR, but do we even use any of the `own`-labeled functions?